src/pre_defined_classes/mathematical_program/hard/q3/MainTest.java

```diff
@@ -0,0 +1,27 @@
+package pre_defined_classes.mathematical_program.hard.q3;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons with distractors
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The area of the sector is: "),
+                new DoubleLiteral("area")
+        };
+    }
+
+    public void runMain() {
+        AreaSector.main(new String[0]);
+    }
+
+    @Test
+    public void mathTest() {
+        assertEquals(24.801128670839425, Double.parseDouble(getItemByName("area")), 0.001, "Your calculated sector area is incorrect.");
```

Comment:
Do we really need all of this specificity? Wouldn't it make sense to limit it to 3 decimal values after the decimal place?
So this would be `24.801` instead and the delta will account for the variation that is smaller than that.